### PR TITLE
BAH-1497 | Add. create new lab role and map to lab-lite privilege

### PR DESCRIPTION
**Pull request by amankrayush**
co-autored by Priyanka Ajmera<pajmera@thoughtworks.com>

### CODE REVIEW

File: openmrs/apps/home/extension.json [hunk 1]
```diff
@@ -129,6 +129,15 @@
     "icon": "fa fa-calendar",
     "order": 14,
     "requiredPrivilege": "app:appointments"
+  },
+  "labLite": {
+    "id": "bahmni.lab",
+    "extensionPointId": "org.bahmni.home.dashboard",
+    "type": "link",
+    "translationKey": "MODULE_LABEL_LAB_ENTRY_KEY",
+    "url": "http://localhost:8200/lab/home",
```
Comment (gsluthra) [inline openmrs/apps/home/extension.json:138]: This URL is incorrect. Points to local. Needs to be "relative" to current server. This config gets deployed on QA/Demo environments (not just on local).

File: openmrs/migrations/addLabRoleWithPrivilege.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="Lab-Role-20220421" author="Priyanka, Aman">
+        <preConditions onFail="CONTINUE">
+            <sqlCheck expectedResult="0">SELECT COUNT(*) FROM role where role = 'lab'</sqlCheck>
```
Comment (gsluthra) [inline openmrs/migrations/addLabRoleWithPrivilege.xml:11]: Roles use Capital Case for the first letter. So this role should be called: "Lab" (not lab). See: https://demo.mybahmni.org/openmrs/admin/users/role.list

**Comment by gsluthra** [comment]
@amankrayush : Please sign the CLA (one time activity). Ensure your git config commit email address is the same one as your github user email ID.. else even if u sign the CLA, it won't work.. since commit email address will be different than the email u signed CLA with.

File: openmrs/apps/home/extension.json (hunk 1 above)
Comment (amankrayush) [reply openmrs/apps/home/extension.json]: resolved with the relative url : "/lab/home"

File: openmrs/migrations/addLabRoleWithPrivilege.xml (hunk 2 above)
Comment (amankrayush) [reply openmrs/migrations/addLabRoleWithPrivilege.xml]: Done

**Review by gsluthra — APPROVED** [review]

**Review by N0-man — APPROVED** [review]

**Pull request merged**